src/framework/ui/internal/platform/linux/linuxplatformtheme.cpp

```diff
@@ -72,3 +91,23 @@ void LinuxPlatformTheme::applyPlatformStyleOnAppForTheme(const ThemeCode&)
 void LinuxPlatformTheme::applyPlatformStyleOnWindowForTheme(QWindow*, const ThemeCode&)
 {
 }
+
+void LinuxPlatformTheme::processSettingChange(const QString& group, const QString& key, const QDBusVariant& value)
+{
+    if (group == "org.freedesktop.appearance" && key == "color-scheme") {
```

Comment:
In other places you're using QStringLiteral, maybe that would be good here too?
```suggestion
    if (group == QStringLiteral("org.freedesktop.appearance") && key == QStringLiteral("color-scheme")) {
```